lib/js/dom.nim

```diff
@@ -9,7 +9,7 @@
 
 ## Declaration of the Document Object Model for the `JavaScript backend
 ## <backends.html#backends-the-javascript-target>`_.
-
+include "system/inclrtl"
```

Comment:
Why is this being added?

Reply:
Its not needed and should be removed

Reply:
since

Reply:
`since`

Reply:
I know it includes `since` but you shouldn't have to manually include it.

Reply:
you need to otherwise you get CT error; but if we merged https://github.com/nim-lang/Nim/pull/11865 we could import it instead of including it

Reply:
Once again putting it out there that I consider this `since` thing to be a waste of time.